### ISSUE TITLE
Logo?

### ISSUE THREAD

**Issue by josephholsten** [sous-chefs/meta#26]
How should we go about getting a logo?

Any particular usages or requirements we should have?

**Comment by juliandunn** [comment]
So I checked with Chef marketing and we don't really have the bandwidth to be able to take this on now... sorry.

**Comment by josephholsten** [comment]
I wasn't banking on it. But it's been mentioned a couple times, so we probably want one.

Thanks for checking tho!

**Comment by rex** [comment]
@juliandunn Thank you for giving it a shot anyway! Seriously appreciated.

**Comment by rex** [comment]
**Alright. I said I wasn't a good designer, and I stand by my word.**

In the interest of having something official, even though it's temporary until we get a _real_ designer to do a logo, I hashed something together for the Brigade's consideration. 

## `v0.0.1`

![chef-brigade-logo-v0 0 1](https://cloud.githubusercontent.com/assets/4247754/8741469/e01e8afe-2c1e-11e5-81c8-aa729dff061a.png)

## `v0.0.2`

![chef-brigade-logo-v0 0 2](https://cloud.githubusercontent.com/assets/4247754/8741474/f0f3bcd2-2c1e-11e5-9c06-f83f2351b4a4.png)

I'm open to commentary but **since I'm not a designer** I have limited capacity for non-constructive criticism. Suggest improvements, provide different images, or come up with a logo of your own and let's talk about it!

**Comment by josephholsten** [comment]
Leave the words out for the moment.

See also: https://portlandflag.files.wordpress.com/2014/09/2013_gfbf_final_web.pdf

**Comment by rex** [comment]
Great feedback. 

## `v0.0.3`

![chef-brigade-logo-v0 0 3](https://cloud.githubusercontent.com/assets/4247754/8742028/7dbadd7e-2c26-11e5-8c91-5fbe70a6d9ef.png)

## `v0.0.3-icon`
- **24px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="24" height="24" />
- **32px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="32" height="32" />
- **48px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="48" height="48" />
- **64px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="64" height="64" />
- **96px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="96" height="96" />
- **128px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="128" height="128" />
- **256px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="256" height="256" />
- **512px**
  - <img src="https://cloud.githubusercontent.com/assets/4247754/8744701/e65a8fba-2c3e-11e5-9156-d740642bbfb5.png" width="512" height="512" />

**Comment by zuazo** [comment]
It is a logo, not a flag. You can include text in a logo. But it has to be square for GitHub.

The one we have now is fine for _brigade_. The one you are proposing is good for _chef_. But I feel that both are missing something.

But I'm not a logo designer / lover. I do not care much about the logo.

Thank you for your proposal, @rex.

**Comment by josephholsten** [comment]
@zuazo <3

**Comment by rex** [comment]
@zuazo Thanks for the feedback :) A few things:

1) Seriously, thank you for the constructive criticism.
2) I'm not a designer either, as is evident by these images. Just wanted to get _something_ as our logo/favicon even though it's going to change as soon as we can get a real designer.
3) The idea behind the multiple chef hats was simply to indicate a multitude of chefs. A _brigade_ of chefs if you will. I'm sure there's a million different ways to translate this into an image, but in the interest of keeping the image simple I felt like it looked okay. 
4) I'm going to post a version with the multiple hats overlaid by the text because I agree with you - logos can have text in them. I'm curious to see how it looks. 
5) The favicon. It's going to be so small that I felt having multiple chef hats like the larger logo would be just too small to see so I kept it at one. 

In short: I **really** can't wait for a designer to contribute a logo :)

**Comment by josephholsten** [comment]
some usage ideas:
- favicon
- header of the marketing site
- avatar in gitub &c
- t-shirt
- READMEs, ideally by combining the brigade logo with the program's logo.

so some nice to haves:
- square form
- high contrast (aka stencil) form: on light bg, on dark
- combining/encapsulating form
- horiz logotype form, usually like 

```
[x] Chef Brigade
```
- vert logotype form, usually like

```
    [x]
Chef Brigade
```

While picking this, we'll probably also want to select:
- forground, background and accent colors
- typeface

@rex: your multi-[toque](https://en.wikipedia.org/wiki/Toque) is well suited to all of these ;-)

**Comment by zuazo** [comment]
@rex, rereading my previous message, sorry to sound so harsh and critical. That was not my intention at all. Drawing a _brigade_ in a logo is not an easy task.

Anyway, I like your hat-brigade. For now, is the best we have. I agree with @josephholsten here.

**Comment by rex** [comment]
@zuazo I didn't take you as being harsh or negatively critical :) Not at all. 

A real designer will hopefully be able to translate _brigade_ into a nice visual. Once I get home tonight, though, I'll mock a few more things up using the multi-toque concept. Side note: I didn't even know what a 'toque' was until @josephholsten linked the wiki article.

**Comment by damacus** [comment]
@tas50 any progress with this one?
